perun-web-gui/src/main/java/cz/metacentrum/perun/webgui/tabs/registrartabs/EditFormItemTabItem.java

```diff
@@ -628,13 +675,25 @@ public void onChange(ChangeEvent event) {
 
 			row++;
 
+			Label srcAttrLabel = new Label("Source attribute:");
+			ft.setWidget(row, 0, srcAttrLabel);
+			ft.setWidget(row, 1, perunSourceAttributeListBox);
+			ftf.setColSpan(row, 1, 2);
+
+			row++;
+			ft.setHTML(row, 1, "Select attribute, which will be used to pre-fill form value. You can select also VO "+(forGroup ? "and group" : "")+" attributes.");
```

Comment:
If forGroup is false, then there will be two spaces between words "VO" and "attributes". I would replace it by:
```
ft.setHTML(row, 1, "Select attribute, which will be used to pre-fill form value. You can select also VO "+(forGroup ? "and group " : "")+"attributes.");
```

Reply:
Fixed in new commit.